imgui.cpp

```diff
@@ -951,6 +951,11 @@ CODE
 #include <stdint.h>     // intptr_t
 #endif
 
+#ifdef MCPE_PLATFORM_NX
+#include <iostream>
+#define HOST_MOUNT_NAME "c"
+#endif
+
```

Comment:
the assumption is that it matches the existing host-mount-name and that the host is a "windows c"

Reply:
since it is a define, it can be over-ridden as a compile option